### DISCUSSION TITLE
macOS `Passwords...` menu only triggers on mouse click

### DISCUSSION THREAD

**Discussion by memco** [ghostty-org/ghostty#3617] [Q&A] [answered]
Not sure if this is a Ghostty or mac thing, but if you go to `Edit > AutoFill > Passwords...` you can get a UI to search passwords and input them, but if you use the help menu search for `Passwords...` and hit return or bind a keyboard shortcut in the mac system settings the UI does not trigger. Any ideas if this should work with keyboard or is it mouse only?

**Comment by 00-kat** [comment]
I have not heard of any password manager in Ghostty, and have also explicitly heard that one won't be implemented (except if done with a plugin using a hypothetical future plugin system).  My guess is that macOS adds this to the bar because Ghostty supports secure input or something.  I'm not certain though, as I don't use macOS.

**Comment by taylrfnt** [comment, marked as answer]
OP is likely referring to this context menu:
![image](https://github.com/user-attachments/assets/04a9b69f-cf47-49d3-a81b-7d9b7df4f9a5)

As far as I know, that is a standard/private MacOS API thing, because it shows up in any application where your cursor sits at a text field (except browsers, as far as my testing goes).  Here's that same menu in Messages, because you know, _everyone needs to AutoFill Passwords when texting their best buddy_ (/s). 
![image](https://github.com/user-attachments/assets/1461db5c-57c9-46e9-ae15-28de046faf94)

You can also find this menu option in System Settings, for some reason.  Funnily enough, that AutoFill menu also seems to completely disregard the preferences of a user set in System Settings, because I have the Passwords app entirely disabled for AutoFill and it still shows up per the screenshots above:
![image](https://github.com/user-attachments/assets/c193c2b1-023f-42f1-9535-29aa48c1626f)

Anyway, yes, I can reproduce this, but I can also reproduce this on **every** application that this AutoFill menu item appears in.  Given that, I'm going to cautiously say "not a Ghostty bug" and reserve the right to eat my words later...  

In case anyone cares to see the keybind setting I used to test this, here's how to configure it in System Settings:
![image](https://github.com/user-attachments/assets/00e4e8ee-e883-4b3d-a6be-269c3a9e49a9)
